geode-core/src/main/java/org/apache/geode/internal/statistics/StatisticsRegistry.java

```diff
@@ -136,6 +147,11 @@ public int getStatListModCount() {
     return getStatsList().toArray(new Statistics[0]);
   }
 
+  @Override
+  public Set<String> getMeterWhitelist() {
+    return meterWhitelist;
```

Comment:
Why a get and a create?

Reply:
`createMeterWhitelist()` is a private helper method to move those details out of the constructor. `getMeterWhitelist()` is the way the registry factory finds the